examples/jenkins/jenkins-persistent-template.json

```diff
@@ -113,6 +113,14 @@
                 },
                 "env": [
                   {
+                    "name": "OPENSHIFT_ENABLE_OAUTH",
+                    "value": "anyvalue"
+                  },
+                  {
+                    "name": "OPENSHIFT_ENABLE_REDIRECT_PROMPT",
+                    "value": "anyvalue"
+                  },
+                  {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
```

Comment:
no they still are usable if running outside openshift ... we just agreed to doc around this in openshift-docs ... don't agree on deletion